sites/upsun/src/get-started/stacks/express/add-database.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Add a database"
+title: "Add a database to Express"
```

Comment:
Can we extract everything that is not directly related to Magento in another PR? That would help the reviews.